### PR TITLE
BaoLe - Taking Reading Test Screen - Custom Toolbar

### PR DESCRIPTION
**Pull request by at-baole**
#10

### CODE REVIEW

File: app/src/main/java/vn/asiantech/englishtest/ListQuestionAdapter.kt [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    override fun getItemCount(): Int {
+        return listQuestions.size
+    }
+
+    override fun onBindViewHolder(holder: ListQuestionViewHolder, position: Int) {
+        holder.bindView(listQuestions[position])
+    }
+
+    inner class ListQuestionViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+        fun bindView(listItems: ListQuestionItems) {
```
Comment (at-tainguyen) [inline app/src/main/java/vn/asiantech/englishtest/ListQuestionAdapter.kt]: `ListQuestionItems`, this name is confusing. It should be `ListQuestionItem`

File: app/src/main/java/vn/asiantech/englishtest/ListQuestionFragment.kt [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        return inflater.inflate(R.layout.fragment_list_questions, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        initRecycleView()
+    }
+
+    private fun initRecycleView() {
+        recycleViewListQuestions.setHasFixedSize(true)
```
Comment (at-tainguyen) [inline app/src/main/java/vn/asiantech/englishtest/ListQuestionFragment.kt]: Please using `apply {...}` for `recycleViewListQuestions` to make the source code clearer

File: app/src/main/java/vn/asiantech/englishtest/ListQuestionFragment.kt [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        recycleViewListQuestions.layoutManager = GridLayoutManager(activity, 5 )
+        setData()
+        recycleViewListQuestions.adapter = mListQuestionItems?.let { ListQuestionAdapter(it) }
+    }
+
+    private fun setData() {
```
Comment (at-tainguyen) [inline app/src/main/java/vn/asiantech/englishtest/ListQuestionFragment.kt]: Please add `TODO` for the mock function to mark this function need to update later

**Comment by at-baole** [comment]
Thank you for reviewing my PR. I fixed codes according to your comments and hope it works.

File: app/src/main/java/vn/asiantech/englishtest/takingreadingtest/TakingReadingTestActivity.kt [hunk 4]
```diff
@@ -34,30 +36,35 @@ class TakingReadingTestActivity : AppCompatActivity(), View.OnClickListener {
         })
         btnBackToListTest.setOnClickListener(this)
         btnListQuestions.setOnClickListener(this)
+
+        chronometer.start()
```
Comment (at-tainguyen) [inline app/src/main/java/vn/asiantech/englishtest/takingreadingtest/TakingReadingTestActivity.kt]: Should handle `stop` this `chronometer` at somewhere

File: app/src/main/res/anim/slide_out_top.xml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<set xmlns:android="http://schemas.android.com/apk/res/android" >
+    <translate
+            android:duration="500"
+            android:fromYDelta="0%"
+            android:toYDelta="100%" >
+    </translate>
+</set>
```
Comment (at-tainguyen) [inline app/src/main/res/anim/slide_out_top.xml]: Do not forget add break line here

**Comment by at-baole** [comment]
Thank you for reviewing my PR. I have add a TODO method to stop chronometer.

**Pull request merged**